_topic_maps/_topic_map_rosa.yml

```diff
@@ -88,6 +88,8 @@ Topics:
   File: rosa-mobb-configure-custom-tls-ciphers
 - Name: Verifying Permissions for a ROSA STS Deployment
   File: rosa-mobb-verify-permissions-sts-deployment
+- Name: Configuring a ROSA cluster to pull images from AWS Elastic Container Registry (ECR)
```

Comment:
Just a comment (and it may not matter), but I dont think ROSA is actually every spelled out in full in the doc? B/C of the way these docs are being handled, that may not matter but wanted to mention just in case.